corehq/apps/userreports/views.py

```diff
@@ -266,12 +266,16 @@ def dispatch(self, request, *args, **kwargs):
     @property
     def main_context(self):
         main_context = super(ReportBuilderView, self).main_context
+        allowed_num_reports = allowed_report_builder_reports(self.request)
         main_context.update({
             'has_report_builder_access': has_report_builder_access(self.request),
-            'at_report_limit':
-                number_of_report_builder_reports(self.domain) >= allowed_report_builder_reports(self.request),
-            'report_limit': allowed_report_builder_reports(self.request),
+            'at_report_limit': (
+                number_of_report_builder_reports(self.domain) >= allowed_num_reports
+                and allowed_num_reports is not None
+            ),
+            'report_limit': allowed_num_reports,
             'paywall_url': paywall_home(self.domain),
+            'pricing_page_url': reverse('public_pricing') if settings.ENABLE_PRELOGIN_SITE else "",
```

Comment:
Won't this just be a dead link if `not settings.ENABLE_PRELOGIN_SITE`?  I guess that only happens on single-domain environments.

Reply:
yeah good point. i should just hide that link